tests/unit_tests/test_unclustered_recruitment.py

```diff
@@ -176,9 +176,9 @@ class MockArgs:
         def __init__(self):
             self.kmers: str = kmers
             self.coverage: str = coverage
-            self.assignments: str = assignments_fpath
+            self.binning: str = assignments_fpath
```

Comment:
Need to include a copyright notice and script explanation at the top.

Reply:
Same comments as above